scripts/prepare/bundle.ts

```diff
@@ -92,7 +92,7 @@ const run = async ({ cwd, flags }: { cwd: string; flags: string[] }) => {
         c.platform = platform || 'browser';
         c.legalComments = 'none';
         c.minifyWhitespace = optimized;
-        c.minifyIdentifiers = optimized;
+        c.minifyIdentifiers = false;
```

Comment:
Could you leave a comment, why this change is needed?

Reply:
Yes!

See here: https://circleci.com/gh/storybookjs/storybook/444855

When we build our code for `dist`, we used to compress our variable names.
Turns out when generating variables names like this `_2` is a valid identifier.
But when you try and create a function with that name using `eval` or `new Function()` it throws.

I decided that not compressing variable names would probably be the right choice here, considering this might make debugging way way easier too.